site/docs/index.md

```diff
@@ -5,62 +5,33 @@ title: Home
 
 
 
+## What is Substrait?
 
+Substrait enables interoperability between different implementations of the data compute ecosystem.  This allows one to make the best solution for any problem by choosing the best components.  It doesn't matter if they use different languages or run on different operating systems -- they just work together.
```

Comment:
@EpsilonPrime do you want to say what Substrait the _project_ is, or what Substrait the _format_ is?

As is, it's more describing the project—which is fine but maybe adds a layer of indirection for people who are trying to grok how Substrait fits into a system architecture.

Reply:
Thanks for getting that feedback.  I've updated the PR with this in mind.